tests/templates/test_integration.py

```diff
@@ -303,6 +303,7 @@
            ]
 
 
+@pytest.mark.usefixtures("tape_mode")
```

Comment:
Example of how to use the fixture: Now all tests in the class below are run twice. At this stage they fail a lot.